lib/src/wallet_connect/wallet_connect_address.dart

```diff
@@ -3,36 +3,95 @@ class WalletConnectAddress {
   final int version;
   final Uri bridge;
   final String key;
-  final String raw;
+  late String raw;
+  late Map<String, String> parameters;
 
   WalletConnectAddress._(
     this.topic,
     this.version,
     this.bridge,
     this.key,
-    this.raw,
-  );
+    String? raw,
```

Comment:
I think it would be cleaner to not have `raw` on this constructor since it overrides all the other parameters

Reply:
marked raw as deprecated and updated it to use the toString method